src/main/config/conversion/import/iso19139.ca.HNAP.FGP-to-iso19139.ca.HNAP.xsl

```diff
@@ -102,6 +102,41 @@
     </xsl:copy>
   </xsl:template>
 
+  <!--Add default unclassified as security constraint if missing from metadata xml-->
+  <xsl:template match="gmd:MD_DataIdentification">
+    <xsl:copy copy-namespaces="no">
+      <xsl:choose>
+        <xsl:when test="gmd:resourceConstraints/gmd:MD_SecurityConstraints/gmd:classification/gmd:MD_ClassificationCode">
```

Comment:
May want the try something like the following
`<xsl:if test="not(gmd:resourceConstraints/gmd:MD_SecurityConstraints/gmd:classification/gmd:MD_ClassificationCode)">`
instead of creating an empty condition.

Reply:
Good suggestion. I will try.